tests/api_connexion/endpoints/test_event_log_endpoint.py

```diff
@@ -291,6 +291,29 @@ def test_should_filter_eventlogs_by_when(self, create_log_model, session):
             assert response.status_code == 200
             assert {eventlog["event"] for eventlog in response.json["event_logs"]} == expected_eventlogs
 
+    def test_should_filter_eventlogs_by_included_events(self, create_log_model):
+        for event in ["TEST_EVENT_1", "TEST_EVENT_2", "cli_scheduler"]:
+            create_log_model(event=event, when=self.default_time)
+        response = self.client.get(
+            "/api/v1/eventLogs?included_events=TEST_EVENT_1,TEST_EVENT_2",
+            environ_overrides={"REMOTE_USER": "test_granular"},
+        )
+        assert response.status_code == 200
+        response_data = response.json
+        assert len(response_data["event_logs"]) == 2
```

Comment:
```suggestion
        assert len(response_data["event_logs"]) == 2
        assert {"TEST_EVENT_1", "TEST_EVENT_2"} == {x["event"] for x in response_data["event_logs"]}
```

Oops, forgot this one in the last round.